src/main/java/com/znsio/e2e/tools/Drivers.java

```diff
@@ -621,58 +619,76 @@ private void attachLogsAndCloseDriver (TestExecutionContext context, String key)
         }
     }
 
-    private void closeWebDriver(String key, Driver driver) {
-        String message = "Browser logs for user: " + key;
+    private void closeWebDriver(String key, @NotNull Driver driver) {
```

Comment:
rename key to a more meaningful variable name

Reply:
updated 'key' to 'userPersona'